### PR TITLE
Some fixes for SeqPrep filtering sub-wf

### PR DESCRIPTION
**Pull request by KateSakharova**
- changed filtering before seqprep (from 100bp to 70bp)
- fixed submitted_reads field in qc_summary
- removed *_FASTQ.fasta.gz from output of raw-reads pipeline

### CODE REVIEW

File: workflows/subworkflows/trim_and_reformat_reads.cwl [hunk 1]
```diff
@@ -77,15 +83,15 @@ steps:
...
+      fastq_count: count_overlapped_reads/count
     out: [ fasta ]
 
   clean_fasta_headers:
     when: $(inputs.fastq_count != 0)
     run: ../../utils/clean_fasta_headers.cwl
     in:
       sequences: convert_trimmed_reads_to_fasta/fasta
-      fastq_count: count
+      fastq_count: count_overlapped_reads/count
```
Comment (Vkale1) [inline workflows/subworkflows/trim_and_reformat_reads.cwl:95]: is fastq count needed as input for any steps other than Trimmomatic?

File: workflows/subworkflows/seqprep-subwf.cwl [hunk 2]
```diff
@@ -46,7 +83,7 @@ steps:
       forward_reads: filter_paired/forward_filtered
       reverse_reads: filter_paired/reverse_filtered
       namefile: forward_reads
-    out: [ merged_reads, forward_unmerged_reads, reverse_unmerged_reads ]
+    out: [ merged_reads, forward_unmerged_reads, reverse_unmerged_reads ]  # compressed
```
Comment (Vkale1) [inline workflows/subworkflows/seqprep-subwf.cwl:86]: Will this will be unzipped output?

**Review by Vkale1 — CHANGES_REQUESTED** [review]

File: workflows/subworkflows/seqprep-subwf.cwl (hunk 2 above)
Comment (KateSakharova) [reply workflows/subworkflows/seqprep-subwf.cwl:86]: I copy help here too (not to forget)
```
Usage:
SeqPrep [Required Args] [Options]
NOTE 1: The output is always gziped compressed.
```

File: workflows/subworkflows/trim_and_reformat_reads.cwl (hunk 1 above)
Comment (KateSakharova) [reply workflows/subworkflows/trim_and_reformat_reads.cwl:95]: I'm using it in condition: when: $(inputs.fastq_count != 0)

Comment (Vkale1) [reply workflows/subworkflows/trim_and_reformat_reads.cwl:95]: ah yes! I see this. Thanks

**Review by Vkale1 — APPROVED** [review]

**Comment by KateSakharova** [comment]
merged with rebase -i

**Pull request closed without merging**